habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/InstallDependenciesMojo.java

```diff
@@ -129,13 +127,43 @@ public void doExecute() throws MojoExecutionException, MojoFailureException {
 
         processManagedDependencyMismatches();
 
-        if (StringUtils.isNotEmpty(this.pypiRepoUrl) && this.addPypiRepoAsPackageSources) {
```

Comment:
I: this was really just extracting the code that was there into a method so it can be called for both regular and dev repositories